src/helpers/file-path.test.js

```diff
@@ -1,23 +1,6 @@
 const filePath = require( './file-path' );
 
-jest.mock( 'process', () => ( { cwd: () => 'cwd' } ) );
-
-expect.extend( {
-	toMatchFilePath: ( received, expected ) => {
-		const pass = ( new RegExp( received.replace( /(\\|\/)/g, '(\\\\|/)' ) ) ).test( expected );
-		if ( pass ) {
-			return {
-				message: () => `expected ${ received } not to match file path ${ expected }`,
-				pass: true,
-			};
-		}
-
-		return {
-			message: () => `expected ${ received } to match file path ${ expected }`,
-			pass: false,
-		};
-	},
-} );
+// Depends on custom matchers defined in __tests__/setup.js.
```

Comment:
The path is incorrect, and personally, I would remove that comment altogether. Otherwise, it should also be included in the `presets.test.js` file, plus any other potential future file.